rakelib/bootstrap.rake

```diff
@@ -1,2 +1 @@
-
-task "bootstrap" => [ "vendor:all", "compile:all" ]
+task "bootstrap" => [ "vendor:gems", "vendor:all", "compile:all" ]
```

Comment:
I try to make the gems installation go in the first place. There is still a bit of room for a refactoring to make this cleaner.